gradoop-examples/src/main/java/org/gradoop/benchmark/sna/SocialNetworkAnalyticsExample.java

```diff
@@ -112,7 +112,7 @@ public static void main(String[] args) throws Exception {
     GradoopFlinkConfig conf = GradoopFlinkConfig.createConfig(env);
 
     // read logical graph
-    DataSource source = new IndexedCSVDataSource(INPUT_PATH, conf);
+    DataSource source = new CSVDataSource(INPUT_PATH, conf);
```

Comment:
Why?

Reply:
Changed that because the pattern has been benchmarked with the ldbc_1 and ldbc_10 datasets, which are not indexed. I will revert the change.